ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.16-nightly-x86.yaml

```diff
@@ -54,6 +54,20 @@ tests:
     - chain: openshift-qe-node-density-heavy
     workflow: openshift-qe-installer-gcp-ipi-ovn-etcd-encryption-fips
   timeout: 5h0m0s
+- always_run: false
+  as: cluster-density-v2-etcd-fips-120nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_NODE_REPLICAS: "120"
+      COMPUTE_NODE_TYPE: e2-standard-8
```

Comment:
I think 'e2-standard-4' should be enough. I referred to AWS's 120 nodes config, it used COMPUTE_NODE_TYPE: m5.xlarge (4vCPU)
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml#L47

Reply:
Done